### DISCUSSION TITLE
exporting query strings

### DISCUSSION THREAD

**Discussion by phamdt** [sqlc-dev/sqlc#852] [Q&A]
is there a way to export the queries? if not, is this something people would find useful as part of some config or something? i'm wanting to reuse the queries in different contexts like in tests that use sqlmock which requires the queries to setup expectations.

**Comment by kyleconroy** [comment]
@phamdt We could add an `export_query_strings` option if this would help you out.

**Reply by phamdt** [reply]
@kyleconroy that'd be amazing!
